monitoring/os.go

```diff
@@ -79,7 +79,7 @@ func (c *OSChecker) Check(ctx context.Context, reporter health.Reporter) {
 
 	reporter.Add(&pb.Probe{
 		Checker: osCheckerID,
-		Detail:  fmt.Sprintf("%s %s is not supported, please see https://gravitational.com/docs/pack/#distributions", id, versionID),
```

Comment:
Why did you remove this hint? It seemed useful.

Reply:
because it's used in manifest checks, and i.e. Anypoint manifest won't contain ubuntu as supported OS, and the error message will be misleading as it'll refer end-user to Gravitational docs where we will claim Ubuntu is supported.